prompt-service/src/unstract/prompt_service/main.py

```diff
@@ -776,7 +776,7 @@ def _retrieve_context(output, doc_id, vector_index, answer) -> str:
         if node.score > 0:
             text += node.get_content() + "\n"
         else:
-            app.logger.info("Node score is less than 0. " f"Ignored: {node.score}")
+            app.logger.info("Node score is less than 0. " f"Ignored: {node}")
```

Comment:
@chandrasekharan-zipstack Won't this be an issue? Because we are exposing user's data from the node

Reply:
Good point @Deepak-Kesavan, my intention was to display the node's ID alone for some reference along with the score. Let me change this